irispy/new_sji.py

```diff
@@ -0,0 +1,179 @@
+'''
+This module provides movie tools for level 2 IRIS SJI fits file
+'''
+
+from datetime import timedelta
+
+from ndcube import NDCube
+import numpy as np
+from astropy.io import fits
+import astropy.units as u
+from astropy.wcs import WCS
+from sunpy.time import parse_time
+from sunpy import config
+from irispy import iris_tools
+
+__all__ = ['SJICube']
+
+TIME_FORMAT = config.get("general", "time_format")
+
+# the following value is only appropriate for byte scaled images
+BAD_PIXEL_VALUE = -200
+# the following value is only appropriate for unscaled images
+BAD_PIXEL_VALUE_UNSCALED = -32768
```

Comment:
@tiagopereira Is the scaling always the same for SJI?  If so, we can hard code the ```BAD_PIXEL_VALUE_UNSCALED``` here.  Otherwise we'll have to think of a more flexible solution.